invenio_app_ils/views.py

```diff
@@ -189,6 +189,11 @@ def _get_loans_ui_config():
     ui_config["circulation"]["loanCompletedStates"] = current_app.config.get(
         "CIRCULATION_STATES_LOAN_COMPLETED", []
     )
+
+    ui_config["circulation"]["defaultDuration"] = current_app.config.get(
+        "CIRCULATION_POLICIES").get(
+        "checkout").get("duration_default")(None).days
```

Comment:
is this `(None)` correct?

Reply:
unfortunately, yes all the functions for config retrieving have signatures with an argument, which is unused but I assumed that it might be a reason for that, so I didn't change the functions